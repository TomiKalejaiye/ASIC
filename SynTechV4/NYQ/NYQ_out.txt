x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
-1328 
-1328 
-1328 
-1328 
-1328 
-1328 
-1328 
-1328 
-1258 
-1258 
-1258 
-1258 
-1258 
-1258 
-1258 
-1258 
782 
782 
782 
782 
782 
782 
782 
782 
65 
65 
65 
65 
65 
65 
65 
65 
-1233 
-1233 
-1233 
-1233 
-1233 
-1233 
-1233 
-1233 
2561 
2561 
2561 
2561 
2561 
2561 
2561 
2561 
-3827 
-3827 
-3827 
-3827 
-3827 
-3827 
-3827 
-3827 
4723 
4723 
4723 
4723 
4723 
4723 
4723 
4723 
-4955 
-4955 
-4955 
-4955 
-4955 
-4955 
-4955 
-4955 
4265 
4265 
4265 
4265 
4265 
4265 
4265 
4265 
-2534 
-2534 
-2534 
-2534 
-2534 
-2534 
-2534 
-2534 
-198 
-198 
-198 
-198 
-198 
-198 
-198 
-198 
3665 
3665 
3665 
3665 
3665 
3665 
3665 
3665 
-7399 
-7399 
-7399 
-7399 
-7399 
-7399 
-7399 
-7399 
10748 
10748 
10748 
10748 
10748 
10748 
10748 
10748 
-12984 
-12984 
-12984 
-12984 
-12984 
-12984 
-12984 
-12984 
13382 
13382 
13382 
13382 
13382 
13382 
13382 
13382 
-11385 
-11385 
-11385 
-11385 
-11385 
-11385 
-11385 
-11385 
6712 
6712 
6712 
6712 
6712 
6712 
6712 
6712 
520 
520 
520 
520 
520 
520 
520 
520 
-9740 
-9740 
-9740 
-9740 
-9740 
-9740 
-9740 
-9740 
19903 
19903 
19903 
19903 
19903 
19903 
19903 
19903 
-29548 
-29548 
-29548 
-29548 
-29548 
-29548 
-29548 
-29548 
36875 
36875 
36875 
36875 
36875 
36875 
36875 
36875 
-39864 
-39864 
-39864 
-39864 
-39864 
-39864 
-39864 
-39864 
36296 
36296 
36296 
36296 
36296 
36296 
36296 
36296 
-23592 
-23592 
-23592 
-23592 
-23592 
-23592 
-23592 
-23592 
-2106 
-2106 
-2106 
-2106 
-2106 
-2106 
-2106 
-2106 
49463 
49463 
49463 
49463 
49463 
49463 
49463 
49463 
-150841 
-150841 
-150841 
-150841 
-150841 
-150841 
-150841 
-150841 
640409 
640409 
640409 
640409 
640409 
640409 
640409 
640409 
533600 
533600 
533600 
533600 
533600 
533600 
533600 
533600 
-175065 
-175065 
-175065 
-175065 
-175065 
-175065 
-175065 
-175065 
79975 
79975 
79975 
79975 
79975 
79975 
79975 
79975 
-29729 
-29729 
-29729 
-29729 
-29729 
-29729 
-29729 
-29729 
-1560 
-1560 
-1560 
-1560 
-1560 
-1560 
-1560 
-1560 
20803 
20803 
20803 
20803 
20803 
20803 
20803 
20803 
-30896 
-30896 
-30896 
-30896 
-30896 
-30896 
-30896 
-30896 
33740 
33740 
33740 
33740 
33740 
33740 
33740 
33740 
-31088 
-31088 
-31088 
-31088 
-31088 
-31088 
-31088 
-31088 
24695 
24695 
24695 
24695 
24695 
24695 
24695 
24695 
-16303 
-16303 
-16303 
-16303 
-16303 
-16303 
-16303 
-16303 
7487 
7487 
7487 
7487 
7487 
7487 
7487 
7487 
449 
449 
449 
449 
449 
449 
449 
449 
-6590 
-6590 
-6590 
-6590 
-6590 
-6590 
-6590 
-6590 
10442 
10442 
10442 
10442 
10442 
10442 
10442 
10442 
-11938 
-11938 
-11938 
-11938 
-11938 
-11938 
-11938 
-11938 
11345 
11345 
11345 
11345 
11345 
11345 
11345 
11345 
-9198 
-9198 
-9198 
-9198 
-9198 
-9198 
-9198 
-9198 
6140 
6140 
6140 
6140 
6140 
6140 
6140 
6140 
-2834 
-2834 
-2834 
-2834 
-2834 
-2834 
-2834 
-2834 
-170 
-170 
-170 
-170 
-170 
-170 
-170 
-170 
2465 
2465 
2465 
2465 
2465 
2465 
2465 
2465 
-3859 
-3859 
-3859 
-3859 
-3859 
-3859 
-3859 
-3859 
4332 
4332 
4332 
4332 
4332 
4332 
4332 
4332 
-4029 
-4029 
-4029 
-4029 
-4029 
-4029 
-4029 
-4029 
3172 
3172 
3172 
3172 
3172 
3172 
3172 
3172 
-2053 
-2053 
-2053 
-2053 
-2053 
-2053 
-2053 
-2053 
908 
908 
908 
908 
908 
908 
908 
908 
52 
52 
52 
52 
52 
52 
52 
52 
-722 
-722 
-722 
-722 
-722 
-722 
-722 
-722 
1002 
1002 
1002 
1002 
1002 
1002 
1002 
1002 
-3375 
-3375 
-3375 
-3375 
-3375 
-3375 
-3375 
-3375 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
0 
