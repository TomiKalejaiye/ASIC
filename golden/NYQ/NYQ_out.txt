x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
x 
0 
0 
0 
0 
0 
0 
0 
-50 
-50 
-50 
-50 
-50 
-50 
-50 
-50 
44 
44 
44 
44 
44 
44 
44 
44 
107 
107 
107 
107 
107 
107 
107 
107 
-251 
-251 
-251 
-251 
-251 
-251 
-251 
-251 
158 
158 
158 
158 
158 
158 
158 
158 
-78 
-78 
-78 
-78 
-78 
-78 
-78 
-78 
175 
175 
175 
175 
175 
175 
175 
175 
-105 
-105 
-105 
-105 
-105 
-105 
-105 
-105 
-42 
-42 
-42 
-42 
-42 
-42 
-42 
-42 
233 
233 
233 
233 
233 
233 
233 
233 
-458 
-458 
-458 
-458 
-458 
-458 
-458 
-458 
375 
375 
375 
375 
375 
375 
375 
375 
-455 
-455 
-455 
-455 
-455 
-455 
-455 
-455 
563 
563 
563 
563 
563 
563 
563 
563 
-581 
-581 
-581 
-581 
-581 
-581 
-581 
-581 
217 
217 
217 
217 
217 
217 
217 
217 
208 
208 
208 
208 
208 
208 
208 
208 
-524 
-524 
-524 
-524 
-524 
-524 
-524 
-524 
950 
950 
950 
950 
950 
950 
950 
950 
-1369 
-1369 
-1369 
-1369 
-1369 
-1369 
-1369 
-1369 
1598 
1598 
1598 
1598 
1598 
1598 
1598 
1598 
-1630 
-1630 
-1630 
-1630 
-1630 
-1630 
-1630 
-1630 
1033 
1033 
1033 
1033 
1033 
1033 
1033 
1033 
-371 
-371 
-371 
-371 
-371 
-371 
-371 
-371 
-463 
-463 
-463 
-463 
-463 
-463 
-463 
-463 
1392 
1392 
1392 
1392 
1392 
1392 
1392 
1392 
-2556 
-2556 
-2556 
-2556 
-2556 
-2556 
-2556 
-2556 
3400 
3400 
3400 
3400 
3400 
3400 
3400 
3400 
-3645 
-3645 
-3645 
-3645 
-3645 
-3645 
-3645 
-3645 
3371 
3371 
3371 
3371 
3371 
3371 
3371 
3371 
-2497 
-2497 
-2497 
-2497 
-2497 
-2497 
-2497 
-2497 
917 
917 
917 
917 
917 
917 
917 
917 
1074 
1074 
1074 
1074 
1074 
1074 
1074 
1074 
-3634 
-3634 
-3634 
-3634 
-3634 
-3634 
-3634 
-3634 
5762 
5762 
5762 
5762 
5762 
5762 
5762 
5762 
-7247 
-7247 
-7247 
-7247 
-7247 
-7247 
-7247 
-7247 
7674 
7674 
7674 
7674 
7674 
7674 
7674 
7674 
-7137 
-7137 
-7137 
-7137 
-7137 
-7137 
-7137 
-7137 
5081 
5081 
5081 
5081 
5081 
5081 
5081 
5081 
-1614 
-1614 
-1614 
-1614 
-1614 
-1614 
-1614 
-1614 
-2741 
-2741 
-2741 
-2741 
-2741 
-2741 
-2741 
-2741 
7337 
7337 
7337 
7337 
7337 
7337 
7337 
7337 
-11602 
-11602 
-11602 
-11602 
-11602 
-11602 
-11602 
-11602 
14556 
14556 
14556 
14556 
14556 
14556 
14556 
14556 
-15913 
-15913 
-15913 
-15913 
-15913 
-15913 
-15913 
-15913 
14437 
14437 
14437 
14437 
14437 
14437 
14437 
14437 
-10168 
-10168 
-10168 
-10168 
-10168 
-10168 
-10168 
-10168 
3242 
3242 
3242 
3242 
3242 
3242 
3242 
3242 
5310 
5310 
5310 
5310 
5310 
5310 
5310 
5310 
-14867 
-14867 
-14867 
-14867 
-14867 
-14867 
-14867 
-14867 
23992 
23992 
23992 
23992 
23992 
23992 
23992 
23992 
-30981 
-30981 
-30981 
-30981 
-30981 
-30981 
-30981 
-30981 
34199 
34199 
34199 
34199 
34199 
34199 
34199 
34199 
-32293 
-32293 
-32293 
-32293 
-32293 
-32293 
-32293 
-32293 
24243 
24243 
24243 
24243 
24243 
24243 
24243 
24243 
-10120 
-10120 
-10120 
-10120 
-10120 
-10120 
-10120 
-10120 
-10593 
-10593 
-10593 
-10593 
-10593 
-10593 
-10593 
-10593 
36705 
36705 
36705 
36705 
36705 
36705 
36705 
36705 
-67000 
-67000 
-67000 
-67000 
-67000 
-67000 
-67000 
-67000 
99505 
99505 
99505 
99505 
99505 
99505 
99505 
99505 
-133649 
-133649 
-133649 
-133649 
-133649 
-133649 
-133649 
-133649 
169984 
169984 
169984 
169984 
169984 
169984 
169984 
169984 
-214485 
-214485 
-214485 
-214485 
-214485 
-214485 
-214485 
-214485 
-359964 
-359964 
-359964 
-359964 
-359964 
-359964 
-359964 
-359964 
367229 
367229 
367229 
367229 
367229 
367229 
367229 
367229 
-717292 
-717292 
-717292 
-717292 
-717292 
-717292 
-717292 
-717292 
405475 
405475 
405475 
405475 
405475 
405475 
405475 
405475 
513722 
513722 
513722 
513722 
513722 
513722 
513722 
513722 
-1397469 
-1397469 
-1397469 
-1397469 
-1397469 
-1397469 
-1397469 
-1397469 
228260 
228260 
228260 
228260 
228260 
228260 
228260 
228260 
-139759 
-139759 
-139759 
-139759 
-139759 
-139759 
-139759 
-139759 
-466568 
-466568 
-466568 
-466568 
-466568 
-466568 
-466568 
-466568 
1109715 
1109715 
1109715 
1109715 
1109715 
1109715 
1109715 
1109715 
-737208 
-737208 
-737208 
-737208 
-737208 
-737208 
-737208 
-737208 
-710911 
-710911 
-710911 
-710911 
-710911 
-710911 
-710911 
-710911 
509463 
509463 
509463 
509463 
509463 
509463 
509463 
509463 
-680819 
-680819 
-680819 
-680819 
-680819 
-680819 
-680819 
-680819 
561619 
561619 
561619 
561619 
561619 
561619 
561619 
561619 
548841 
548841 
548841 
548841 
548841 
548841 
548841 
548841 
-1250712 
-1250712 
-1250712 
-1250712 
-1250712 
-1250712 
-1250712 
-1250712 
281659 
281659 
281659 
281659 
281659 
281659 
281659 
281659 
-17673 
-17673 
-17673 
-17673 
-17673 
-17673 
-17673 
-17673 
-384979 
-384979 
-384979 
-384979 
-384979 
-384979 
-384979 
-384979 
1203059 
1203059 
1203059 
1203059 
1203059 
1203059 
1203059 
1203059 
-629180 
-629180 
-629180 
-629180 
-629180 
-629180 
-629180 
-629180 
-639273 
-639273 
-639273 
-639273 
-639273 
-639273 
-639273 
-639273 
632678 
632678 
632678 
632678 
632678 
632678 
632678 
632678 
-616759 
-616759 
-616759 
-616759 
-616759 
-616759 
-616759 
-616759 
684645 
684645 
684645 
684645 
684645 
684645 
684645 
684645 
620555 
620555 
620555 
620555 
620555 
620555 
620555 
620555 
-1141680 
-1141680 
-1141680 
-1141680 
-1141680 
-1141680 
-1141680 
-1141680 
371783 
371783 
371783 
371783 
371783 
371783 
371783 
371783 
70563 
70563 
70563 
70563 
70563 
70563 
70563 
70563 
-274017 
-274017 
-274017 
-274017 
-274017 
-274017 
-274017 
-274017 
1272853 
1272853 
1272853 
1272853 
1272853 
1272853 
1272853 
1272853 
-504319 
-504319 
-504319 
-504319 
-504319 
-504319 
-504319 
-504319 
-577029 
-577029 
-577029 
-577029 
-577029 
-577029 
-577029 
-577029 
757362 
757362 
757362 
757362 
757362 
757362 
757362 
757362 
-545918 
-545918 
-545918 
-545918 
-545918 
-545918 
-545918 
-545918 
792321 
792321 
792321 
792321 
792321 
792321 
792321 
792321 
716082 
716082 
716082 
716082 
716082 
716082 
716082 
716082 
-1064966 
-1064966 
-1064966 
-1064966 
-1064966 
-1064966 
-1064966 
-1064966 
502578 
502578 
502578 
502578 
502578 
502578 
502578 
502578 
110019 
110019 
110019 
110019 
110019 
110019 
110019 
110019 
-106459 
-106459 
-106459 
-106459 
-106459 
-106459 
-106459 
-106459 
1278566 
1278566 
1278566 
1278566 
1278566 
1278566 
1278566 
1278566 
-308586 
-308586 
-308586 
-308586 
-308586 
-308586 
-308586 
-308586 
-590667 
-590667 
-590667 
-590667 
-590667 
-590667 
-590667 
-590667 
956305 
956305 
956305 
956305 
956305 
956305 
956305 
956305 
-514081 
-514081 
-514081 
-514081 
-514081 
-514081 
-514081 
-514081 
-990654 
-990654 
-990654 
-990654 
-990654 
-990654 
-990654 
-990654 
437126 
437126 
437126 
437126 
437126 
437126 
437126 
437126 
-625504 
-625504 
-625504 
-625504 
-625504 
-625504 
-625504 
-625504 
199413 
199413 
199413 
199413 
199413 
199413 
199413 
199413 
759912 
759912 
759912 
759912 
759912 
759912 
759912 
759912 
-1312379 
-1312379 
-1312379 
-1312379 
-1312379 
-1312379 
-1312379 
-1312379 
16406 
16406 
16406 
16406 
16406 
16406 
16406 
16406 
96770 
96770 
96770 
96770 
96770 
96770 
96770 
96770 
-615438 
-615438 
-615438 
-615438 
-615438 
-615438 
-615438 
-615438 
1088958 
1088958 
1088958 
1088958 
1088958 
1088958 
1088958 
1088958 
-414475 
-414475 
-414475 
-414475 
-414475 
-414475 
-414475 
-414475 
-924928 
-924928 
-924928 
-924928 
-924928 
-924928 
-924928 
-924928 
571830 
571830 
571830 
571830 
571830 
571830 
571830 
571830 
-573691 
-573691 
-573691 
-573691 
-573691 
-573691 
-573691 
-573691 
332983 
